src/SDWaveFile.cpp

```diff
@@ -173,14 +173,48 @@ int SDWaveFile::begin()
   return 1;
 }
 
+int SDWaveFile::findData(int offset) {
```

Comment:
Maybe it's clearer if this is renamed to `findDataHeaderOffset`?

Reply:
yes is better now